cmd/apps_cmd.go

```diff
@@ -48,8 +48,17 @@ func AppDeploy(c *cli.Context) error {
 	debugCmdFuncInfo(c)
 	appSvc, formatter := WireUpApp(c)
 
-	checkRequiredFlags(c, []string{"id", "location_id", "cloud_account_id", "hostname"}, formatter)
-	app, err := appSvc.DeployApp(utils.FlagConvertParams(c), c.String("id"))
+	checkRequiredFlags(c, []string{"id", "location-id", "cloud-account-id", "server-plan-id", "hostname"}, formatter)
```

Comment:
The `server-plan-id` flag should not be required

Reply:
Done!